S21/rkarmuri/ConvertToLowerCase.java

```diff
@@ -0,0 +1,98 @@
+//Method to convert given string to lowercase
+package com.company;
+
+import java.util.Locale;//Package to accept multiple languages
+
+public class ConvertToLowerCase {
+    public String convert(String wordSample, String language) {
+        System.out.println("Input word: " + wordSample);
+        System.out.println("Input language tag: " + language);
+        String wordLowerCase = wordSample.toLowerCase();//Conversion in English
+        String trazString = wordSample.toLowerCase(Locale.forLanguageTag("az"));//Conversion in Turkish and Azerbaijani
+        String russString = wordSample.toLowerCase(Locale.forLanguageTag("ru-Latn"));//Conversion in Russian
+        String hungarianString = wordSample.toLowerCase(Locale.forLanguageTag("hu"));//Conversion in Hungarian
+        //Conversion cases for Greek
+        String greekString = wordLowerCase.replace('Σ', 'σ').replace('Ε', 'ε').
+                replace('Α', 'α').replace('Β', 'β').
```

Comment:
Can we have a dot at the beginning of the line instead of the end like this? :
.replace('Α', 'α').replace('Β', 'β')

Reply:
No we cannot have a dot at the beginning.